app/src/main/java/com/hedvig/app/feature/offer/OfferExt.kt

```diff
@@ -0,0 +1,17 @@
+package com.hedvig.app.feature.offer
+
+import com.apollographql.apollo.api.Response
+import com.hedvig.android.owldroid.graphql.OfferQuery
+
+fun Response<OfferQuery.Data>.priceOrNull() = data
```

Comment:
I think we're going to have to switch away from `lastQuoteOfMember` pretty soon - perhaps we can make this extension on `Quote` and sum it up on the affected `Quote`s later on?

Reply:
Yes! I think thats suited another PR though, I'll create an issue for it.